src/EditorFeatures/VisualBasicTest/Diagnostics/QualifyMemberAccess/QualifyMemberAccessTests.vb

```diff
@@ -0,0 +1,358 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis.CodeFixes
+Imports Microsoft.CodeAnalysis.CodeFixes.Qualify
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.Simplification
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeFixes.QualifyMemberAccess
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.Diagnostics.QualifyMemberAccess
+    Public Class QualifyMemberAccessTests
+        Inherits AbstractVisualBasicDiagnosticProviderBasedUserDiagnosticTest
+
+        Friend Overrides Function CreateDiagnosticProviderAndFixer(workspace As Workspace) As Tuple(Of DiagnosticAnalyzer, CodeFixProvider)
+            Return Tuple.Create(Of DiagnosticAnalyzer, CodeFixProvider)(New VisualBasicQualifyMemberAccessDiagnosticAnalyzer(), New VisualBasicQualifyMemberAccessCodeFixProvider())
+        End Function
+
+        Private Function TestAsyncWithOption(code As String, expected As String, opt As PerLanguageOption(Of Boolean)) As Task
+            Return TestAsync(code, expected, options:=[Option](opt, True))
+        End Function
+
+        Private Function TestMissingAsyncWithOption(code As String, opt As PerLanguageOption(Of Boolean)) As Task
+            Return TestMissingAsync(code, options:=[Option](opt, True))
+        End Function
+
+        <WorkItem(7065, "https://github.com/dotnet/roslyn/issues/7065")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)>
+        Public Async Function QualifyFieldAccess_LHS() As Task
+            Await TestAsyncWithOption(
+"Class C : Dim i As Integer : Sub M() : [|i|] = 1 : End Sub : End Class",
+"Class C : Dim i As Integer : Sub M() : Me.i = 1 : End Sub : End Class",
+SimplificationOptions.QualifyFieldAccess)
+        End Function
+
+        <WorkItem(7065, "https://github.com/dotnet/roslyn/issues/7065")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)>
+        Public Async Function QualifyFieldAccess_RHS() As Task
```

Comment:
Good idea.  I've added a test for qualifying the backing field on an auto-property when `QualifyFieldAccess` is true.  I opted not to try to qualify the backing field if only Qualify**Property**Access is true because any developer explicitly accessing the backing field presumably knows what they're doing and would be confused to see a ``Me.`' qualification being suggested.

That, and it would make this feature needlessly complex while only adding minimal value for 2 people.

I've also added a test for accessing `Shared` (not `static`; what do you think this is, C#?) with a '`Me.`' qualifier.  VB doesn't allow '`Me.`' in modules so that's been skipped.

I'll be adding the commit to this PR with the tests shortly.